dependencies.props

```diff
@@ -9,15 +9,15 @@
        These ref versions are pulled from https://github.com/dotnet/versions.
   -->
   <PropertyGroup>
```

Comment:
Did you intend to pull in all the other dependency updates as well?

Reply:
yes, I cherry-picked the commit.